frontend/src/presentation/components/QuestionFilters.tsx

```diff
@@ -131,27 +131,26 @@ export const QuestionFilters: React.FC<QuestionFiltersProps> = ({
                 </div>
 
                 <div className={styles.filterItem}>
-                <Select
-                    placeholder={QUESTIONS_FILTER_TEXT.SELECT_DIFFICULTY}
-                    value={selectedDifficulty}
-                    onChange={handleDifficultyChange}
-                    className={styles.difficultyFilter}
-                    optionLabelProp="label"
-                    options={[{
-                        value: DIFFICULTY_TEXT.ALL,
-                        label: <span>{DIFFICULTY_TEXT.ALL}</span>
-                    },{
-                        value: DIFFICULTY_TEXT.EASY,
-                        label: <span style={{ color: getDifficultyColor(DIFFICULTY_TEXT.EASY) }}>{DIFFICULTY_TEXT.EASY}</span>
-                    },{
-                        value: DIFFICULTY_TEXT.MEDIUM,
-                        label: <span style={{ color: getDifficultyColor(DIFFICULTY_TEXT.MEDIUM) }}>{DIFFICULTY_TEXT.MEDIUM}</span>
-                    },{
-                        value: DIFFICULTY_TEXT.HARD,
-                        label: <span style={{ color: getDifficultyColor(DIFFICULTY_TEXT.HARD) }}>{DIFFICULTY_TEXT.HARD}</span>
-                    }
-                    ]}
-                />
+                    <Select
+                        placeholder="Choose Difficulty"
+                        value={selectedDifficulty === DIFFICULTY_TEXT.ALL ? 'Choose Difficulty' : selectedDifficulty}
+                        onChange={handleDifficultyChange}
+                        className={styles.difficultyFilter}
+                        optionLabelProp="label"
+                        options={[{
+                            value: DIFFICULTY_TEXT.ALL,
```

Comment:
You can use the difficultyOptions I've defined in QuestionUtils.  Then, append it with { value: "All", label: "All" }. Is that better or nah

Reply:
Cannot because your difficultyOptions' value field is QuestionDifficulty type, but "All" isn't because you also commented you don't like it. I'll thus leave it like this and remove the "all" type in QuestionDifficulty.